### PR TITLE
XIVY-12427 Add vscode connection

### PR DESCRIPTION
**Pull request by ivy-lli**
Connection for https://github.com/axonivy/vscode-extensions/pull/102

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ivy-lmu) [inline .project:1]: why is this addition needed?

**Review by ivy-lmu — APPROVED** [review]
👍

File: .project (hunk 1 above)
Comment (ivy-lli) [reply .project:1]: Used that the dev-wf-ui can be imported as module in the designer
image

**Review by ivy-cst — COMMENTED** [review]
Cool, we should add this also for the redirect site like this :+1:
